packages/testing/test/test.spec.ts

```diff
@@ -98,7 +97,7 @@ describe('Test the testkit', () => {
       onValidate: jest.fn().mockReturnValue(undefined),
     };
 
-    const testkit = createTestkit([plugin1, enableIf(false, plugin2)], createSchema());
+    const testkit = createTestkit([plugin1, false && plugin2], createSchema());
```

Comment:
We need to make our types more powerful so we can allow this. Not sure how to do it nicely though cc @n1ru4l @ardatan @dotansimha